src/naovoce/settings/base.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please do not add `-*- coding: utf-8 -*-` stub, because Python 3 has utf-8 as default.